### PR TITLE
feat(evergreen theme): fix visually incorrect tokens

### PR DESCRIPTION
**Pull request by serifluous**
Compared the Evergreen theme in Figma with code + Twilio theme in Figma, and caught some bugs in code:

- Updated $color-text-icon-brand-highlight to segment green, instead of twilio red
- Darkened $color-background-overlay to match what's in Figma
- Fixed typo in $color-background-primary-strongest where a `O` should have been a `0`.
- Matched $color-text-user to twilio theme (instead of default theme) making it purple, instead of dark gray

### CODE REVIEW

File: packages/paste-design-tokens/tokens/themes/evergreen/global/border-color.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ props:
 
   # primary action borders
   color-border-primary-strongest:
-    value: "#OC1E56"
+    value: "#0C1E56"
```
Comment (TheSisb) [inline packages/paste-design-tokens/tokens/themes/evergreen/global/border-color.yml:24]: Good catch lol

**Review by TheSisb — APPROVED** [review]

File: packages/paste-design-tokens/tokens/themes/evergreen/global/border-color.yml (hunk 1 above)
Comment (nkrantz) [reply packages/paste-design-tokens/tokens/themes/evergreen/global/border-color.yml:27]: :hahaa:

File: .changeset/few-pumpkins-perform.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/design-tokens': patch
+'@twilio-paste/core': patch
+---
+
+Fixed visual bugs in Evergreen theme tokens: $color-text-icon-brand-highlight, $color-background-overlay, $color-background-primary-strongest, $color-text-user
```
Comment (nkrantz) [inline .changeset/few-pumpkins-perform.md:6]:
```suggestion
[Design tokens] Fixed visual bugs in Evergreen theme tokens: $color-text-icon-brand-highlight, $color-background-overlay, $color-background-primary-strongest, $color-text-user
```

suggestion: we usually write the package name in the description for readability

**Review by nkrantz — DISMISSED** [review]

File: .changeset/few-pumpkins-perform.md (hunk 2 above)
Comment (serifluous) [reply .changeset/few-pumpkins-perform.md]: done, thanks!

**Pull request merged**